python/lsst/ip/isr/isrTask.py

```diff
@@ -295,6 +292,11 @@ class IsrTaskConfig(pexConfig.Config):
         default=True,
         doc="Load and use transmission_atmosphere (if doAttachTransmissionCurve is True)?"
     )
+    doEmpiricalReadNoise = pexConfig.Field(
+        dtype=bool,
+        default=False,
+        doc="Calculate empirical read noise instead of value from AmpInfo data?"
```

Comment:
Clarify: "instead of using value from amp.getReadNoise?" This also makes me wonder what sets the read noise value in the amplifier info in the first place. Where does it usually come from?

Reply:
The read noise is hard-wired into the camera configuration, often in the form of entries in FITS files.